CommonMark/common.py

```diff
@@ -1,12 +1,12 @@
-from __future__ import absolute_import, unicode_literals
+
 
 import re
 import sys
 
 try:
     from urllib.parse import quote
 except ImportError:
-    from urllib import quote
+    from urllib.parse import quote
```

Comment:
this change doesn't make sense - the whole point is to import `quote` from `urllib` if `url.parse` fails, i.e., we are on python 2.